pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -130,7 +130,10 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="System.Net.Http">
-      <Version>4.1.3</Version>
+      <Version>4.1.4</Version>
+    </NETCoreApp10Dependency>
+    <NETCoreApp10Dependency Include="System.Net.Http.WinHttpHandler">
```

Comment:
This one shouldn't be needed as it isn't referenced directly by Microsoft.NETCore.App.